python/send.py

```diff
@@ -370,6 +381,14 @@ def getJobInfo(argv):
         if instatus < num_jobs:
             num_jobs = instatus
             warning("Directory contains only " + str(instatus) + " files, using all for the reconstruction")
+
+    if args.addPileupToSignal: 
```

Comment:
Please try to combine it with lines 395-397. It should be fairly similar with the exception of running once again getInputFiles. Also it misses the comments to explain why this additional step is needed.